overlord/devicestate/systems.go

```diff
@@ -347,8 +347,9 @@ func createSystemForModelFromValidatedSnaps(
 
 	wOpts := &seedwriter.Options{
 		// RW mount of ubuntu-seed
-		SeedDir: boot.InitramfsUbuntuSeedDir,
-		Label:   label,
+		SeedDir:                    boot.InitramfsUbuntuSeedDir,
+		Label:                      label,
+		IgnoreOptionFileExtentions: true,
```

Comment:
do unit tests here need any change to show the new setup (like remove extensions?) ?